waf.tf

```diff
@@ -25,7 +25,7 @@ resource "azurerm_application_gateway" "waf" {
 #    )}"
 
   lifecycle {
-    ignore_changes = ["http_listener", "request_routing_rule", "probe", "backend_http_settings","ssl_certificate","authentication_certificate"]
+    ignore_changes = ["http_listener", "request_routing_rule", "probe", "backend_address_pool", "backend_http_settings","ssl_certificate","authentication_certificate"]
```

Comment:
😆 you guys are still using this?

Reply:
Well, nothing can change for us until we go live ...

Reply:
😢